lib/validator.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../lib/player'
+
+module Validator
+    include PlayerType
+
+    def position_valid?(input:, board:) 
+        int = input.to_i
+        return board_range_valid?(int,board) && board.position_empty?(int)
+    end
+
+    def board_range_valid?(int, board)
+        int >= 0 && int < board.max_positions
+    end
+    
+    def board_size_valid?(input:)
```

Comment:
Maybe this function should use info from `Board` to determine whether the board size is valid